VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:


BO_ 1330 update_data_message: 8 Vector__XXX
 SG_ message_type M : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ update_length m0 : 8|32@1+ (1,0) [0|0] "" Vector__XXX
 SG_ update_md5 m2 : 16|32@1+ (1,0) [0|0] "" Vector__XXX
 SG_ data_block_index m1 : 8|24@1+ (1,0) [0|0] "" Vector__XXX
 SG_ update_data m1 : 32|32@1+ (1,0) [0|0] "" Vector__XXX
 SG_ update_md5_idx m2 : 8|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 1331 update_progress_message: 4 Vector__XXX
 SG_ update_block_idx : 0|24@1+ (1,0) [0|0] "" Vector__XXX
 SG_ received_len : 24|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ received_md5 : 25|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ written : 26|1@1+ (1,0) [0|0] "" Vector__XXX



VAL_ 1330 message_type 0 "UpdateStart" 1 "UpdateData" 2 "Md5" ;

